README.md

```diff
@@ -204,16 +204,17 @@ Klaviyo.createEvent(event)
 
 ### Setup
 The Klaviyo Push SDK for Android works as a wrapper around `FirebaseMessagingService`, so the
-setup process is very similar to the Firebase client documentation linked above.  
-In your `AndroidManifest.xml` file, register `KlaviyoPushService` to receive `MESSAGING_EVENT` intents.
+setup process is very similar to the Firebase client documentation linked above.  We've added it to the library
+manifest, which will be merged with your app on build. If you'd like to override this implementation with your 
+own push service, consider it will impact SDK functionality.
```

Comment:
similar to below: this is already called out in the advanced setup area, so I'd just omit this here.